### ISSUE TITLE
LCD reverts to normal post power up screen during prints over an hour - V16.08.2

### ISSUE THREAD

**Issue by RetromanIE** [TinkerGnome/Ultimaker2Marlin#47]
This issue was happening on my Ultimaker 2 Extended, and I have now upgraded it to a Extended + and am using the + firmware and getting the same issue

When printing I have the normal Tinker display screen with all the info and option to change speed/temp/flow etc. After an hour or two it reverts to the normal screen you see once the Ultimaker is booted up (with an option to print/pre heat/change settings etc). The print will continue fine, and I can make changes in the advanced menu to temp/flow/bed temp, but the info screen during the print does not come back until the next print job. Seems to be an intermittent issue as it doesn't always happen.

As I mention this happens on my Ultimaker 2 Extended (which is now an Extended+).

**Comment by Turtletrumpet** [comment]
Hello,

I print with Octoprint and i had today a similar problem. After 12 hours of printing the display doesn't go back to "main menu" after finished the printjob. It's the first time with V16.08.2 that this happen.
After power cycle all works normal.

regards,

turtle

**Comment by RetromanIE** [comment]
Yes I also have that issue since this morning (on top of the other issue). After the print I need to power off the printer to gain access to the main menu again. Not using Octoprint though, just regular print from SD.

**Comment by Turtletrumpet** [comment]
Hello,

one possible hint to find the bug in the firmware:
I turn on the autooff of the LCD-display after 40 mintues. I do not use this feature before!
Normally my LCD-display is always on.

regards,

turtle

**Comment by EldRick** [comment]
Same as an issue I reported recently. I do not turn the LCD off.

**Comment by foreachthing** [comment]
Maybe random. This print (~3 hours), the screen reverted.
A print over 9 hours was fine.

**Comment by Turtletrumpet** [comment]
Hello,

I compiled the 16.08.2 for U2E+ with the compiler Arduino V1.8.1. I used this [link](https://ultimaker.com/en/community/38154-how-to-build-marlin-the-way-tinkerghome-and-ultimaker-do-it-if-you-have-windows) to setup the IDE. Since 3 prints with 18 to 21 hours, the problem , that  the display doesn't go back to "main menu" after finished the printjob, doesn't happen. I upload the created hex-file with cura 15.04.06. I use simplify3d as slicer and octprint. I don't use sdcard.

regards,

turtle

**Comment by gr5** [comment]
@Turtletrumpet I strongly recommend 16.01

**Comment by Turtletrumpet** [comment]
@gr5 Thank you for the recommendation! And sorry, that I have misjudged you in the past. My fault.

regards,

turtle
